libparsec/crates/platform_mountpoint/tests/unit/operations/windows_winfsp_tests.rs

```diff
@@ -63,7 +63,6 @@ async fn ok(tmp_path: TmpPath, env: &TestbedEnv) {
                         "-create_related_test",
                         // TODO: investigate misc tests
                         "-create_notraverse_test",
-                        "-delete_mmap_test",
```

Comment:
Why we remove the `mmap` test ?

Reply:
Because this test is now passing \o/

(the `-delete_mmap_test` should be read here "disable test `delete_mmap_test`")

Reply:
T_T